source/common/common/posix/thread_impl.h

```diff
@@ -82,7 +82,7 @@ class PosixThreadFactory : public ThreadFactory {
    * Creates a new generic thread from the specified `thread_routine`. When the
    * thread cannot be created, this function will crash.
    */
-  ThreadPtr createThread(std::function<void()> thread_routine, OptionsOptConstRef options);
+  ThreadPtr createThread(std::function<void()> thread_routine, OptionsOptConstRef options) override;
```

Comment:
Crazy that this was not breaking before! How does C++ even work?!

Reply:
yeah I would've expected it to be a compiler warning at least, and for our toolchain to be configured to fail on such a warning.  Pretty dangerous!